src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx

```diff
@@ -113,29 +113,34 @@ Here's more detail about the two authentication domain sections:
 
 ### Managing users [#managing-users]
 
-For how users are added and managed, you can select **Manually** or **Identity provider**. 
+For how users are added and managed, you can select **Manually** or **Identity provider**. The option to use your identity provider to provision users is available only if your organization has [Pro or Enterprise edition](https://newrelic.com/pricing). 
 
-The option to use your identity provider to provision users is available only if your organization has [Pro or Enterprise edition](https://newrelic.com/pricing). If you choose **Identity provider**, you must follow [additional steps for using specific identity providers](/docs/accounts/accounts/automated-user-management/automated-user-provisioning-single-sign). Once you do that, these users are created on the new user model and synced in New Relic based on your identity provider configuration. 
+If you choose **Identity provider**, **you must follow the [steps for automated user management](/docs/accounts/accounts/automated-user-management/automated-user-provisioning-single-sign#how-to)**. Once you completely those steps return to the user migration wizard and docs. 
 
-Note that if you're already using an app made by Okta, Azure, or OneLogin, you'll probably need to use a newer version made for New Relic's newer user model. For more on this, see [Identity provider apps](#idp-apps).
+Some tips and recommendations for doing this step: 
 
-After you complete provisioning users, you can confirm that these users have been added by seeing if they’re in the new [**User management** UI](#manage-users).
+* If you're already using a New Relic app for either Okta, Azure, or OneLogin, you're likely using an out-of-date one. The out-of-date apps are titled "New Relic by account" while the newer apps are titled "New Relic by organization."
+* Once you complete those steps, new user records are created on the new user model and synced in New Relic based on your identity provider configuration. In a later step (step #6), you'll migrate your users' assets to these new user records. 
+* After you complete provisioning users, you can confirm that these users have been added by seeing if they’re in the new [**User management** UI](#manage-users).
+* To access the new New Relic user management UI, you'll have to ensure you're logged in via your new user record: this may require logging out, logging back in, and verifying your email to see all logins associated with that email. 
+* For the step of granting access to groups in New Relic: for some basics of how access grants work, see [Access grants](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts/#understand-concepts). For a tutorial on how to create access grants, see the [user management tutorial](/docs/accounts/accounts-billing/new-relic-one-user-management/tutorial-add-new-user-groups-roles-new-relic-one-user-model/#group-access).
```

Comment:
A little unclear what the second bullet point is trying to convey. I think it should be fine to cut "For the step of granting access" and just include "For some basics about how access grants work"?

Alternatively, you can format it like this:

* To manage your granting access:
      - See our Access grants doc for basics
      - See our user management tutorial to create access grants

Or something similar.